### ISSUE TITLE
FR: Ability to view Exif data removed/remaining

### ISSUE THREAD

**Issue by Gitoffthelawn** [szTheory/exifcleaner#167]
I really like how you show how much Exif data was originally stored in a file, and how much is remaining after ExifCleaner performs its processing.  Allowing the user to view the before and after data (perhaps in a tooltip) would be a huge and welcome improvement.

Thanks for your consideration.

**Comment by szTheory** [comment]
Do you mean like a diff view where the exif data that was removed in the after tooltip, but but with a strikethrough ~like this~ so you can easily tell what was removed?

**Comment by Gitoffthelawn** [comment]
Yes, that would be a good way to do it.  You could also use different colors (red for removed), but then you potentially add accessibility issues for those with certain types of colorblindness.

**Comment by szTheory** [comment]
Good idea! This might take me a while to get around to, but I agree it would be a large usability improvement so I'd like to add it.

**Comment by Gitoffthelawn** [comment]
Sounds great.  Because of the current inexplicable outgoing TCP connection, I'm likely going to hold off from using ExifCleaner until that is resolved (or we at least understand what is causing it).  So that will give you some time to work on this! :) :) :) 😄
